src/Microsoft.Diagnostics.EventFlow.Inputs.DiagnosticSource/Microsoft.Diagnostics.EventFlow.Inputs.DiagnosticSource.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.6;netstandard2.0;net451;net471</TargetFrameworks>
```

Comment:
I do not see anything specific to net471 in your code. Do we need to target 4.7.1 specifically, or is net451 sufficient?

(we do have to target both netstandard1.6 and netstandard2.0 because of compatibility issues between .NET Core 1.1 and 2.x)